codecov.yml

```diff
@@ -1,6 +1,12 @@
+# github_checks:
+#     annotations: false  # remove annoying inline code comments
 coverage:
   status:
     project: off
     patch:
       default:
+        # informational: true  # turn this one on to get cov tests to always pass
         target: 100%
+comment:
+  behavior: new  # delete old comment and post new
+  require_changes: true  # comment only if the coverage changes
```

Comment:
I would prefer to always see the comment, so that I know that the coverage has been checked and is OK when I'm reviewing something. Then I can immediately see that I do not need to worry about that and I do not have to go through the list of changed files to check that indeed no code was changed or maybe code was changed but codecov integration is somehow broken?

Reply:
OK, I find the coverage map and table confusing and distracting (not really well thought as a dashboard), and I'd like to see it only if the change I made introduced a change in the coverage, up to you guys

Reply:
I kind of agree with you, @valeriupredoi, in that I also have not quite wrapped my head around that map. But I would prefer to keep it for a bit just to be exposed to it to see if I learn something from it.

Reply:
OK sounds good, am gonna comment that line out as well, this PR is getting more extinct than Manny the mammoth :rofl: